GooglePlayInstant/Editor/PlayInstantEditorMenu.cs

```diff
@@ -23,76 +23,85 @@ namespace GooglePlayInstant.Editor
+        // Note: cannot use string.Format() in attribute arguments below.
+        private const string PlayInstant = "PlayInstant/";
+        private const string PlayInstantDocumentation = PlayInstant + "Documentation/";
+
+        [MenuItem(PlayInstantDocumentation + "GitHub Project", false, 100)]
         private static void GitHubProject()
         {
             Application.OpenURL("https://github.com/google/play-instant-unity-plugin");
         }
 
-        [MenuItem("PlayInstant/Documentation/Developer Site", false, 101)]
+        [MenuItem(PlayInstantDocumentation + "Developer Site", false, 101)]
         private static void OpenDevDocumentation()
         {
             Application.OpenURL("https://g.co/InstantApps");
         }
 
-        [MenuItem("PlayInstant/Documentation/Digital Asset Links", false, 102)]
+        [MenuItem(PlayInstantDocumentation + "Digital Asset Links", false, 102)]
         private static void VerifyAndroidAppLinks()
         {
             Application.OpenURL("https://developer.android.com/training/app-links/verify-site-associations#web-assoc");
         }
 
-        [MenuItem("PlayInstant/Documentation/Open Source License", false, 103)]
+        [MenuItem(PlayInstantDocumentation + "Open Source License", false, 103)]
         private static void OpenSourceLicense()
         {
             Application.OpenURL("https://www.apache.org/licenses/LICENSE-2.0");
         }
 
-        [MenuItem("PlayInstant/Report a Bug", false, 110)]
+        [MenuItem(PlayInstant + "Report a Bug", false, 110)]
         private static void OpenReportBug()
         {
             Application.OpenURL("https://github.com/google/play-instant-unity-plugin/issues");
         }
 
-        [MenuItem("PlayInstant/Build Settings...", false, 200)]
+        [MenuItem(PlayInstant + "Build Settings...", false, 200)]
         private static void OpenEditorSettings()
         {
             BuildSettingsWindow.ShowWindow();
         }
 
-        [MenuItem("PlayInstant/Player Settings...", false, 201)]
+        [MenuItem(PlayInstant + "Player Settings...", false, 201)]
         private static void CheckPlayerSettings()
         {
             PlayerSettingsWindow.ShowWindow();
         }
 
-        // Note: cannot use string.Format() in an attribute argument.
-        [MenuItem("PlayInstant/Set up " + PlayInstantRunner.InstantAppsSdkShortDisplayName + "...", false, 202)]
+        [MenuItem(PlayInstant + "Set up " + PlayInstantRunner.InstantAppsSdkShortDisplayName + "...", false, 202)]
         private static void SetUpPlayInstantSdk()
         {
             PlayInstantRunner.InstallPlayInstantSdk();
         }
 
-        [MenuItem("PlayInstant/Quick Deploy...", false, 203)]
+        [MenuItem(PlayInstant + "Quick Deploy...", false, 203)]
         private static void QuickDeployOverview()
         {
             QuickDeployWindow.ShowWindow();
         }
-        
+
+        [MenuItem(PlayInstant + "Check for Plugin Updates...", false, 204)]
```

Comment:
This feels like it belongs in a different section than "Build Settings..." or "Quick Deploy...". ie. those are for using the plugin, but this one is for managing the plugin. Maybe put it below "report a bug"?

Reply:
You're right, that's much better! Done.